src/app/main/media/gallery/gallery-media/gallery-media.component.html

```diff
@@ -53,5 +47,14 @@
       </o-column-collapsible>
 
     </div>
-  </div>
+    <div content>
```

Comment:
Its necessary showing into a flex direction `column` because by default the content is proyected into a flex direction `row`
![image](https://user-images.githubusercontent.com/32088627/229105682-90520195-9978-42ef-825a-3055e5d657f8.png)